submissions/adebayoObafemi/database/init.sql

```diff
@@ -0,0 +1,60 @@
+/* create the database for the task */
+create database foodbag;
+
+/* show the databases to confirm the creation of the intende database */
+show databases;
+
+/* select the database for use */
+USE foodbag;
+
+/* create the food_list table */
+create table food_list (
+	id int not null auto_increment,
+    food_type varchar(20) not null,
```

Comment:
Hello @bayooba First the food table should contain its id and type. There are four different food categories on the homepage that would be added to this table. All these columns are required.

Change line 13 to:  type ENUM('fast_food', 'vegetable_fruits', 'drinks_cocktails', 'restaurants') NOT NULL DEFAULT 'fast_food'